javascript/ql/lib/semmle/javascript/Actions.qll

```diff
@@ -54,11 +103,59 @@ module Actions {
     Workflow getWorkflow() { result = workflow }
   }
 
+  /** An environment variable in 'env:' */
+  abstract class Env extends YamlNode, YamlString {
+    /** Gets the name of this environment variable. */
+    abstract string getName();
+  }
+
+  /** A workflow level 'global' environment variable. */
+  class GlobalEnv extends Env {
```

Comment:
Minor: Slightly prefer `WorkflowEnvVariable` since we talk about jobs and steps later, which are also Actions concepts.